addenda98_test.go

```diff
@@ -82,14 +84,10 @@ func BenchmarkAddenda98String(b *testing.B) {
 func testAddenda98ValidRecordType(t testing.TB) {
 	addenda98 := mockAddenda98()
 	addenda98.recordType = "63"
-	if err := addenda98.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda98.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
return the fieldError	

if addenda98.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda98.recordType)
	}